logpush.go

```diff
@@ -20,6 +19,8 @@ type LogpushJob struct {
 	LogpullOptions     string             `json:"logpull_options"`
 	DestinationConf    string             `json:"destination_conf"`
 	OwnershipChallenge string             `json:"ownership_challenge,omitempty"`
+	MaxUploadBytes     int                `json:"max_upload_bytes,omitempty"`
```

Comment:
neither of these are documented publicly so we'll need to get that addressed before merging.